andela/travel_tool_back
# 160714903 view accomodation details

#### What does this PR do?
Enable user to view accommodation details as part of the request modal

#### Description of Task to be completed?
Upon login and making a successful request on Travela, when a user wants to view/confirm the details of the request they have made, they should view the accommodation details as well. Similarly, when a manager logins in to approve a user's requests, they should see the accommodation arrangement of the user.

#### How should this be manually tested?

- Clone the repo
- Change into the project directory
- Check out this branch with git checkout ft-view-accomodation-details-160714903
- Run yarn db:rollmigrate to seed the DB
- On postman: Make a new GET request to retrieve a particular request using the url `http://127.0.0.1:5000/api/v1/requests/<requestID>`
You should retrieve request details that include trip details of a requester, together with bed, room and guesthouse details

#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
[160714903](https://www.pivotaltracker.com/story/show/160714903)
#### Screenshots (if appropriate)
![image](https://user-images.githubusercontent.com/37342753/46346138-ae130c00-c64f-11e8-966e-232dfdd5ec1a.png)
![image](https://user-images.githubusercontent.com/37342753/46346234-09dd9500-c650-11e8-87a6-28dab11b57ac.png)


#### Questions:
N/A
